tsconfig.base.json

```diff
@@ -10,7 +10,7 @@
     "emitDecoratorMetadata": true,
     "skipLibCheck": true,
     "types": ["node", "jest"],
-    "lib": ["es2018"],
+    "lib": ["es2019"],
```

Comment:
Just curious, why did this change?

Reply:
I'm using `flatMap` which is only available from `es2019`. I could definitely use something like `reduce`, but `flatMap` reads better.